deploy/olm-catalog/kubefed-operator/4.2/image-references

```diff
@@ -2,6 +2,10 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
+  - name: ose-kubefed
```

Comment:
So this change is indeed required!?

Reply:
Yes, sorry. I know more now than I did before.